scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -1,5 +1,5 @@
 <form name="editForm">
-    <div sd-modal data-model="vocabulary" class="modal--x-large">
+    <div sd-modal data-model="vocabulary" class="modal--x-large" ng-if="vocabulary.field_type !== 'upload' ">
```

Comment:
Is this `ng-if` still needed after components were moved to react?

Reply:
Good catch, we don't need it.